resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,428 @@
+        - in: path
+          name: departureId
+          description: The ID specifying the departure.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: abcdef1234567890
+      requestBody:
+        description: JSON with message type
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/TestCustomsTransitsMovementsDeparturesMessagesRequest'
+              - description: JSON with message type
+                example:
+                  message:
+                    messageType: IEXXX
+            example:
+              message:
+                messageType: IEXXX
+        required: true
+      responses:
+        '201':
+          description: When a test message has been successfully injected.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/TestCustomsTransitsMovementsDeparturesMessagesResponse'
+                - description: JSON Payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/departures/1234567890abcdef/messages/1234567890abcdef
+                      departure:
+                        href: /customs/transits/movements/departures/1234567890abcdef
+                    departureId: 1234567890abcdef
+                    messageId: 1234567890abcdef
+                    messageType: IEXXX
+                    body: <CCXXX>...</CCXXX>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/departures/1234567890abcdef/messages/1234567890abcdef
+                  departure:
+                    href: /customs/transits/movements/departures/1234567890abcdef
+                departureId: 1234567890abcdef
+                messageId: 1234567890abcdef
+                messageType: IEXXX
+                body: <CCXXX>...</CCXXX>
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/BadRequest'
+                  - description: The supplied JSON payload was invalid.
+                    example:
+                      code: BAD_REQUEST
+                      message: The supplied JSON payload was invalid.
+              example:
+                code: BAD_REQUEST
+                message: The supplied JSON payload was invalid.
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/NotFound'
+                  - description: The specified departure does not exist, has been archived or is not available to the EORI number.

```

Comment:
Supplied departure not found or does not exist or has been archived or is not available to the EORI number.